src/components/views/context_menus/RoomGeneralContextMenu.tsx

```diff
@@ -45,14 +45,50 @@ import { useSettingValue } from "../../../hooks/useSettings";
 
 export interface RoomGeneralContextMenuProps extends IContextMenuProps {
     room: Room;
+    /**
+     * Called when the 'favourite' option is clicked, after the handler is executed.
```

Comment:
uhh what's a handler?

Reply:
I was using it generally to refer to whatever code it normally executes, but I guess it's a buit misleading. Reworded to hopefully be simpler.